ebos/eclgenericproblem.cc

```diff
@@ -373,8 +373,15 @@ updateNum(const std::string& name, std::vector<T>& numbers)
 
     unsigned numElems = gridView_.size(/*codim=*/0);
     numbers.resize(numElems);
+    const auto num_regions = eclState_.getTableManager().getTabdims().getNumSatTables();
```

Comment:
Do we ever call `updateNum()` on anything other than `SATNUM` or `IMBNUM`?  Because if we do, this is the wrong upper bound.